cardano-node/src/Cardano/CLI/Parsers.hs

```diff
@@ -208,8 +210,8 @@ parseGenesisParameters =
     <*> parseFakeAvvmOptions
     <*> parseLovelacePortionWithDefault
           "avvm-balance-factor"
-          "AVVM balances will be multiplied by this factor (defaults to 1)."
-          1
+          "AVVM balances will be multiplied by this factor (defaults to 1e-15)."
+          (toRational ((1 :: Int) % 1e15))
```

Comment:
@dcoutts changed exported terms in `Cardano.Chain.Common.LovelacePortion`.  This gives us only access to transform `LovelacePortion` to / from `Rational`. @Jimbo4350 is it fine for you to pass a rational on the command line?
It makes easier to pass large values but more difficutl to pass smaller ones; e.g. now `0.1` and `0.000000000000001` instead of `100000000000000` or `1` .